src/api/app/models/architecture.rb

```diff
@@ -23,12 +23,13 @@ class Architecture < ApplicationRecord
   validates :name, presence: true
 
   #### Class methods using self. (public and then private)
-
+  # rubocop:disable Rails/IndexBy
```

Comment:
@krauselukas, what about update the `rubocop_todo.yml` file instead of disabling the cop manually.

Reply:
:+1: done